src/common.ts

```diff
@@ -661,8 +659,9 @@ export function fieldFromJSON(
     case "f32":
     case "f64":
     case "string":
-    case "bytes":
       return `${paramPrefix}${ty.name}`
+    case "bytes":
+      return `Buffer.from(${paramPrefix}${ty.name})`
```

Comment:
Shouldn't this be `Uint8Array.from(<>)`?

Reply:
yes